packages/flame/test/effects2/rotate_effect_test.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/src/effects2/rotate_effect.dart';
+import 'package:flame/src/effects2/standard_effect_controller.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('RotateEffect', () {
```

Comment:
It would be nice to have some test that covers a longer timespan or `isInfinite`, and `isAlternating` (is it still called that?) to see that it always hits the correct angle for min and max on the curve when it has run as a lone effect on a component for a longer while.

Reply:
I've added a test for an effect running for 1000 seconds at ~100fps, and the observed numeric discrepancy at the end is on the order of 1e-12 to 1e-11.